### ISSUE TITLE
Future diverge from original gruvbox

### ISSUE THREAD

**Issue by sQVe** [ellisonleao/gruvbox.nvim#93]
👋🏼,

I've been using this plugin for quite some time now and it's been great, so 🙏🏼 for all the work put into it. 

One thing that gives me a bit of **FOMO** is some of the styling options of other themes like https://github.com/folke/tokyonight.nvim or https://github.com/rebelot/kanagawa.nvim. I feel that keeping `gruvbox.nvim` aligned, option-wise, with the original gruvbox is a missed opportunity since it would be quite easy to make it a lot more configurable. Especially since there's a lot of different `gruvbox` projects out there, mainly due to performance and lack of customization.

What are you guys thoughts on ☝🏼?

**Comment by roland-rollo** [comment]
That's a lovely idea. But project lacks man power now. Maybe if create organization and transfer there this repo, more people could work at gruvbox.nvim. @ellisonleao probably have problem with time, so even with PR from other peoples, it still is blocked by one creator time. Especially, lua plugins and neovim options rapidly evolve. For this moment, lush is no-no, and there is PR for removing it, but it waits for approval. I personally don't have time and skills to scratch it more (it's more time consuming than I thought). And don't know, for @ellisonleao would think about let other people to project or org.

**Comment by stvhuang** [comment]
Agree with @roland-rollo. Maybe we can start with @mhdahmad' [port](https://github.com/mhdahmad/gruvbox.nvim).

**Comment by ellisonleao** [comment]
Hey everyone, I know it's frustrating to wait for something to get merged and new features to come up, but I am definitely listening to everyone's interests on keeping this repo as good as it should be. I am currently on vacation but I am already reviewing the lush removal and it should be the first thing that will come in. After that I am already thinking in a new config system to allow a lot of customization. I am not against having more ports or new orgs because that's the way open source is, but if you can just a little bit we can get this done. Thanks!

**Comment by roland-rollo** [comment]
Thank you for info. I  will wait.

**Comment by ellisonleao** [comment]
hey @sQVe going back to this discussion, do you have any proposal for a config system in place? I would like to keep a minimal config but supporting all the current settings. Would love to hear you guys thoughts, maybe in another issue

**Comment by roland-rollo** [comment]
New topic/issue is good idea for config. @ellisonleao could you create it? It would be easier to list propositions and what idea you approve and what no.

Little off topic. Is this part good?
```
vim.cmd([[colorscheme gruvbox]])
```
Shouldn't it be?
```
vim.cmd("colorscheme gruvbox")
```
Or it's the same?

**Comment by ellisonleao** [comment]
They do the same thing, the first one is just a lua syntax for multiline strings

**Comment by ellisonleao** [comment]
created #96 and i am not closing this one
